esmvaltool/cmorizers/obs/cmorize_obs_cru.py

```diff
@@ -63,7 +63,7 @@ def _unzip(filepath, out_dir):
     return new_path
 
 
-def cmorization(in_dir, out_dir, cfg):
+def cmorization(in_dir, out_dir, cfg, **kwargs):
```

Comment:
```suggestion
def cmorization(in_dir, out_dir, cfg, _):
```
since you're never using the user config anywhere